src/NHibernate/Logging.cs

```diff
@@ -272,36 +358,158 @@ public bool IsEnabled(InternalLogLevel logLevel)
 		}
 	}
 
+	/// <summary>
+	/// Base class for <see cref="INHibernateLogger"/> implementations, provides <see cref="ITransitionalInternaLogger"/>
+	/// methods.
+	/// </summary>
+	// Since 5.1
+	[Obsolete("To be used as logger base class for implementing the obsolete interface ITransitionalInternaLogger until it gets dropped.")]
+	public abstract class NHibernateLoggerBase
```

Comment:
This is a base class meant to be drop once we drop the old logger. A bit special way of handling obsolete, but this allows implementors of the new interface to only implement its two new methods. The base class implements all the obsolete ones (and the five "semi-dummy" ones). When it will be dropped, implementors will just need to cease inheriting the base class.